_data/meltano/utilities/dagster/quantile-development.yml

```
@@ -0,0 +1,51 @@
+name: dagster
+label: Dagster
+domain_url: https://dagster.io/
+namespace: dagster
+pip_url: dagster-ext
+ext_repo: https://github.com/quantile-development/dagster-ext
+maintenance_status: development
+repo: https://github.com/dagster-io/dagster
+executable: dagster_invoker
+logo_url: /assets/logos/utilities/dagster.png
+variant: quantile-development
+commands:
+  describe:
+    executable: dagster_extension
+    args: describe
+    description: Describe the Dagster Extension.
+  init:
+    args: initialize
+    executable: dagster_extension
+    description: Initialize the Dagster extension.
+  up:
```

@JulesHuisman another thing I noticed is that we use `ui` in other places like [superset](https://github.com/meltano/hub/blob/33b8d89a3ec0868ff8b5a12a6411b1a9a5c2adbd/_data/meltano/utilities/superset/apache.yml#L41) and [airflow](https://github.com/meltano/hub/blob/33b8d89a3ec0868ff8b5a12a6411b1a9a5c2adbd/_data/meltano/utilities/airflow/apache.yml#L32) for launching frontend services. Would it make sense to follow that convention here too?

cc @tayloramurphy any thoughts?

---

```suggestion
  start:
```

@aaronsteers actually suggested we start moving towards the `systemd`  terms which I think makes sense https://github.com/meltano/hub/pull/984#discussion_r1010603159